docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+## Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or aliases. 
+Aliases are short names TestCafe assigns to browsers for quick reference. 
+A list of all the available aliases can be obtained by the `testcafe --list-browsers` command (see [below](#b)).
+
+The following example demonstrates how to run a test in two browsers: Internet Explorer and Google Chrome.
+
+```sh
+testcafe ie,chrome c:\Tests\SampleFixture.test.js
+```
+
+### All Browsers
+
+To run tests against all installed browsers, use the *all* alias.
+
+```sh
+testcafe all c:\Tests\SampleFixture.test.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify *remote* as a browser alias. 
+
+```sh
+testcafe remote c:\Tests\SampleFixture.test.js
+```
+
+TestCafe will provide a URL that you need to open in the desired browser on your remote device. 
+After that, your remote browser will be connected to TestCafe and tests will start.
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote c:\Tests\SampleFixture.test.js
+```
+ 
+TestCafe will provide an individual URL for each browser.
+Note that tests will start only after all the browsers are connected.
+
+## File Path/GLOB Pattern
+
+Specify one or more file paths to run tests from. If you specify several files, separate them by a space.
+
+```sh
+testcafe ie c:\Tests\Fixture1.test.js c:\Tests\Fixture2.test.js
+```
+
+You can use globbing patterns to specify a set of files. 
+For example, the following command runs tests from files that matches *c:\Tests\&#42;Page&#42;.js*, namely, *c:\Tests\ExamplePage.test.js* and *c:\Tests\MainPage.test.js*.
```

Comment:
> c:\Tests\&#42;Page&#42;.js

Legacy HTML-encoded value here?

Use code (ticks - `) to highlight: paths, code blocks, opts, etc. here and after.